azure-functions/acbs-function/Dockerfile

```diff
@@ -14,5 +14,6 @@ WORKDIR /home/site/wwwroot
 COPY package.json .
 
 RUN npm i
+RUN npm cache clean --force
```

Comment:
Out of interest, why is this needed?

Reply:
This is to reclaim the disk space, and requires `--force` flag.
Reference: https://docs.npmjs.com/cli/v7/commands/npm-cache